docs/enterprise-guide/serialization.md

```diff
@@ -32,6 +30,7 @@ If you're looking to do a one-time migration from H2 to MySQL/Postgres, then use
 - User accounts or settings
 - Alerts on saved questions
 - Personal Collections or their contents (except for the user specified with the `--user` flag; see below)
+- Revision history of questions and dashboards (so questions and dashboards loaded via serialization won't have "last edited by" and "last edited at" property)
```

Comment:
```suggestion
```
Again, I don't think I docs should list all the known bugs - otherwise we should have 600+ additions to our docs.
Something that are clear limitations should of course be listed, but not bugs.
And this one has always been a problem, but just never got noticed, since this metadata wasn't very prominent until lately.